scripts/integration-helpers/framework.ts

```diff
@@ -214,6 +217,9 @@ export class TriggerEndToEndTest {
         this.storageV2FinalizedTriggerCount++;
       }
       if (data.includes(STORAGE_FUNCTION_V2_METADATA_LOG)) {
+        console.log("storage md trigger v2");
+        console.log(data);
+        console.log("---");
```

Comment:
Leaving these and other debug statements here for now while we come to a consensus on what the integ tests trigger values should be, I'll remove them all before a merge